### DISCUSSION TITLE
Production Deploy not indexing

### DISCUSSION THREAD

**Discussion by Valenmetal** [vercel/community#4062] [Help] [answered]
### Summary

This is my first time using Vercel, and i want to use the "valentinogeronzi.vercel.app" domain to host my site. Everything is working just fine but i can't find my site on the browser. I tried with robots.txt but i dont know if used it correctly or not, or if there is something im missing. I use Vite/React btw.
Thanks!

Repo: https://github.com/Valenmetal/Portfolio

### Steps to Reproduce

Repo: https://github.com/Valenmetal/Portfolio

**Comment by amyegan** [comment]
Make sure you're using a [production](https://vercel.com/docs/deployments/environments#production) build (e.g. vercel --prod from the CLI, or pushing to main branch). [Preview deployments are not indexed](https://vercel.com/guides/are-vercel-preview-deployment-indexed-by-search-engines) on purpose, so you'll need to make sure you're using a production deployment.

**Reply by Valenmetal** [reply]
Amy, thanks for your quick response! indeed its a production build, i forgot to say that. But it still not indexing :(

**Comment by z4nr34l** [comment, marked as answer]
@Valenmetal I've checked your website deployment as it's public, and you are missing most simple SEO things.

Add `sitemap.xml` and `robots.txt` to your project, and later post that sitemap URL to Google Search Console or any other search provider of your choice.

Remember that currently most search engines are slowly indexing new websites, sometimes it's taking even months if you are not sending URLs to index manually using Google Indexing API or IndexNow protocol.

Hope it will help you understand basic SEO requirements, that will help engines index your website faster.

If you have any questions feel free to ask.

Happy coding!

**Reply by Valenmetal** [reply]
Thanks! I've done everything. My web it is now indexed, my sitemap it's not being accepted by google search console tho. And the icon it's not loading for a reason.

**Reply by Robin0048** [reply]
Can you share what have you did to make it indexed?
